### DISCUSSION TITLE
Order of notes

### DISCUSSION THREAD

**Discussion by Arcovoltaire** [reorproject/reor#104] [Ideas]
Currently the left bar updates the order for the last changed note. There could be an option to put the order as the creation date. And the possibility of creating notes in their appropriate folder, currently it is created outside the folder by default and we have to drag it to it.

**Comment by samlhuillier** [comment]
Yes this seems like a fairly straightforward improvement to make. We can get to this in the next week or so - but perhaps you'd be interested in helping us by implementing this in the codebase?
